src/Views/AccountView.vala

```diff
@@ -205,9 +205,9 @@ public class Installer.AccountView : AbstractInstallerView {
             return true;
         } else {
             if (username_is_taken) {
-                username_error_revealer.label = _("Username is already taken");
+                username_error_revealer.label = _("Chosen username already taken");
             } else if (!username_is_valid) {
-                username_error_revealer.label = _("Username can only contain lowercase letters and numbers, without spaces");
+                username_error_revealer.label = _("A username can only contain lowercase letters and numbers, without spaces");
```

Comment:
This should be "A username must only contain ...".

Reply:
You would have to say "A username must contain no other characters than letters and numbers, without the use of spaces in between" for that to work.

Reply:
To me (and I am English) the meaning is the same.